.dockerignore

```diff
@@ -1,4 +1,7 @@
 node_modules
 npm-debug.log
 .idea
-.elasticbeanstalk
\ No newline at end of file
+.elasticbeanstalk
```

Comment:
I misread this change as you were removing these directories vs adding them to the ignore.  You are absolutely correct!